src/cljs/witan/ui/components/dashboard/data.cljs

```diff
@@ -25,12 +23,25 @@
   [selected-id _]
   (route/navigate! :app/datapack-create))
 
-(defn filter-fn
-  [file-type-filter]
-  (case file-type-filter
-    :files (comp (partial = "stored") :kixi.datastore.metadatastore/type)
-    :datapacks (comp (partial = "datapack") :kixi.datastore.metadatastore/bundle-type)
-    (constantly true)))
+(defn search-bar
+  [{:keys [on-search]}]
+  (let [current-search-value (route/get-query-param :search-term)
+        current-metadata-filter (or (route/get-query-param :metadata-filter)
+                                    "Everything")]
+    [:div.search-bar.flex-vcenter
+     [:div.flex.search-input
+      (shared/search-filter (get-string :string/search) on-search {:current-search-value current-search-value})]
+     [:div.flex.search-dropdown
+      (icons/filter-list :small)
+      [:select {:id  "metadata-filter"
+                :type "text"
+                :value current-metadata-filter
+                :placeholder nil
+                :on-change #(let [selected (.. % -target -value)]
+                              (route/assoc-query-param-drop-page! :metadata-filter selected)
+                              (controller/raise! :search/dashboard (route/get-query-map)))}
+       (for [metadata-filter ["Everything" "Files" "Datapacks"]]
```

Comment:
I'd like `["Everything" "Files" "Datapacks"]` to be a var somewhere, and for it to use the strings file. (Imagine one day if they want to deploy in a different language)